roles/network_plugin/flannel/handlers/main.yml

```diff
@@ -9,7 +9,10 @@
   notify:
     - Flannel | reload systemd
     - Flannel | reload docker.socket
+    - Flannel | reconfigure docker restart behavior (atomic)
```

Comment:
A comment would help explain the ordering of docker daemon config updates and restarts.